src/Interpreters/InterpreterSystemQuery.cpp

```diff
@@ -435,6 +436,16 @@ BlockIO InterpreterSystemQuery::execute()
             break;
 #else
             throw Exception(ErrorCodes::SUPPORT_IS_DISABLED, "The server was compiled without the support for Parquet");
+#endif
+        }
+        case Type::DROP_OBJECT_STORAGE_LIST_OBJECTS_CACHE:
```

Comment:
Is list caching works only on Parquet files or generally on any S3 ListObject requests?

Reply:
Ah, copy and paste issues. Should be any :D

Reply:
Done